apiserver/Dockerfile

```diff
@@ -1,3 +1,4 @@
+ARG CALICO_BASE
```

Comment:
Does it still work if you move this arg closer to where it is being used (i.e., above the `FROM ${CALICO_BASE}` line)?

Reply:
No, for some reason :thinking: It seems the scoping of the arg works strangely for `FROM` clauses